rllib/utils/replay_buffers/prioritized_replay_buffer.py

```diff
@@ -114,9 +114,9 @@ def sample(
 
         Args:
             num_items: Number of items to sample from this buffer.
-            beta: To what degree to use importance weights
-                (0 - no corrections, 1 - full correction).
-            **kwargs: Forward compatibility kwargs.
+            beta: To what degree to use importance weights (0 - no corrections,
+            1 - full correction).
```

Comment:
+4 indent ->

Reply:
Same as above! Either this one or +8 works for sphinx. :/